src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Extensions/SymbolExtensions.cs

```diff
@@ -62,7 +62,7 @@ private static bool HasVisibleConstructor(ITypeSymbol type, bool includeInternal
             {
                 foreach (IMethodSymbol constructor in namedType.Constructors)
                 {
-                    if (!constructor.IsStatic && constructor.IsVisibleOutsideOfAssembly(includeInternals))
+                    if (!constructor.IsStatic && constructor.IsVisibleOutsideOfAssembly(includeInternals, includeEffectivelySealedSymbols: true))
```

Comment:
Although the containing type is effectively sealed, would you describe the members that way too? I feel like `includeEffectivelyPrivateSymbols` is more accurate, however a user won't really know what that means unless we explain it. So I guess this is fine.